### PR TITLE
remove useless code.

### PR DESCRIPTION
**Pull request by hanfei1991**
@shenli @coocood @zimulala @tiancaiamao
PTAL, say goodbye to old plan :)
#1608

### CODE REVIEW

File: executor/explain_test.go [hunk 1]
```diff
@@ -15,84 +15,78 @@ package executor_test
...
 
 func (s *testSuite) TestExplain(c *C) {
-	plan.UseNewPlanner = false
-	defer testleak.AfterTest(c)()
-	tk := testkit.NewTestKit(c, s.store)
-	tk.MustExec("use test")
-	tk.MustExec("drop table if exists t1, t2")
-	tk.MustExec("create table t1 (c1 int primary key, c2 int, index c2 (c2))")
-	tk.MustExec("create table t2 (c1 int unique, c2 int)")
+	//defer testleak.AfterTest(c)()
```
Comment (shenli) [inline executor/explain_test.go]: Can we remove the test case? Or add a TODO here?

Comment (hanfei1991) [inline executor/explain_test.go]: adding it to todo list is better

Comment (shenli) [inline executor/explain_test.go]: OK! You can add a TODO before the function.

**Comment by coocood** [comment]
LGTM

**Comment by shenli** [comment]
LGTM

**Comment by shenli** [comment]
@hanfei1991 Have you passed all the tidb-test?

**Comment by hanfei1991** [comment]
@shenli of course!

**Pull request merged**